easybuild/easyconfigs/b/bliss/bliss-0.77_install_fix.patch

```diff
@@ -0,0 +1,31 @@
+diff -u CMakeLists.txt.orig CMakeLists.txt
```

Comment:
@PetrKralCZ Please add a short description at the top of the patch + mention author or source (and fix checksum accordingly)